util/tousd.py

```diff
@@ -51,6 +52,10 @@ def nftvolsToUsd(
     cleancase.assertNFTvols(nftvols)
     cleancase.assertRates(rates)
 
+    # Add native token rates
+    for chainID in WRAPPED_TOKEN_ADDRS.keys():
+        symbols[chainID][WRAPPED_TOKEN_ADDRS[chainID]] = WRAPPED_TOKEN_SYMBOLS[chainID]
```

Comment:
There's too much going on in a single line. It takes too much effort to decipher. Something as simple as this should be breezy easy.

I recommend to break it into 2-3 lines. Compute the addr on one line. Compute symbol on another. And on the third, update the dict.